### PR TITLE
Iterate on charts

### PR DESCRIPTION
**Pull request by jckr**
Refined the code for charts to make it less trivial, updated documentation accordingly.

### CODE REVIEW

File: demos/add-charts/charts.js [hunk 1]
```diff
@@ -3,26 +3,32 @@ import {charts} from './style';
 
 import {XYPlot, VerticalBarSeries, XAxis, YAxis
 } from 'react-vis';
-
 export default function Charts({pickups}) {
+  console.log(pickups);
```
Comment (abmai) [inline demos/add-charts/charts.js]: Delete?

File: demos/line-charts/app.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ import {tooltipStyle} from './style';
 
 const MAPBOX_STYLE = 'mapbox://styles/uberdata/cive485h000192imn6c6cc8fc';
 // Set your mapbox token here
-const MAPBOX_TOKEN = process.env.MAPBOX_ACCESS_TOKEN; // eslint-disable-line
+// const MAPBOX_TOKEN = process.env.MAPBOX_ACCESS_TOKEN; // eslint-disable-line
+const MAPBOX_TOKEN = 'pk.eyJ1IjoiamNrciIsImEiOiJjaXJnbHVzMnkwMTZkZzZucmdhdWo5aGFlIn0.X5jPZV3EvmIB01r2bMBjsg';// process.env.MAPBOX_ACCESS_TOKEN; // eslint-disable-line
```
Comment (abmai) [inline demos/line-charts/app.js:13]: This token shouldn't be public right? Or is this a public token we're using?

Comment (jckr) [reply demos/line-charts/app.js:13]: i updated the diff removing this line

File: demos/add-charts/charts.js (hunk 1 above)
Comment (jckr) [reply demos/add-charts/charts.js]: THANKS!!!

**Pull request merged**